Assets/Scripts/Framework/Utils/ObjectParenting/ObjectParentUtils.cs

```diff
@@ -0,0 +1,28 @@
+using UnityEngine;
+
+using DerailedDeliveries.Framework.ParentingSystem;
+
+namespace DerailedDeliveries.Framework.Utils.ObjectParenting
+{
+    /// <summary>
+    /// An util class for object parenting.
+    /// </summary>
+    public static class ObjectParentUtils
+    {
+        /// <summary>
+        /// Tries to get the <see cref="ObjectParent"/> of the given <see cref="GameObject"/>. This will also look in parents.
```

Comment:
```suggestion
        /// Tries to get the <see cref="ObjectParent"/> of the given <see cref="GameObject"/>. This will also look over all connected parents until the root is reached.
```

This would make it more clear.